hmpps_cpr_splink/cpr_splink/model/model_files/model_2025_03_17_1e09.json

```diff
@@ -299,6 +303,34 @@
                 }
             ],
             "comparison_description": "CustomComparison"
+        },
+        {
+            "output_column_name": "manual_override",
+            "comparison_levels": [
+                {
+                    "sql_condition": "manual_override_l IS NULL OR manual_override_r IS NULL",
+                    "label_for_charts": "manual_override_r is NULL",
+                    "is_null_level": true
+                },
+                {
+                    "sql_condition": "manual_override_l = manual_override_r",
+                    "label_for_charts": "Exact match on manual_override",
+                    "m_probability": 0.99999999999999999999999999999999999999999999999999,
```

Comment:
See https://github.com/moj-analytical-services/splink/pull/2758
You have to get to 1e-323 before you get problems (double precision).

1e-50 chosen because it's big enough to drown anything the model could conceivably return - it corresponds to a match weight of 166.  Highest positive score observed in practice is about 60.